widgetssdk/src/main/java/com/glia/widgets/view/snackbar/SnackBarDelegate.kt

```diff
@@ -50,10 +51,10 @@ internal abstract class SnackBarDelegate(
 
     fun show() = snackBar.show()
 
-    private fun makeSnackBar(view: View, titleStringKey: Int, stringProvider: LocaleProvider, snackBarTheme: SnackBarTheme?): Snackbar {
+    private fun makeSnackBar(view: View, @StringRes titleStringKey: Int, stringProvider: LocaleProvider, snackBarTheme: SnackBarTheme?): Snackbar {
         val bgColor = snackBarTheme?.backgroundColorTheme?.primaryColor ?: view.getColorCompat(fallbackBackgroundColor)
         val textColor = snackBarTheme?.textColorTheme?.primaryColor ?: view.getColorCompat(fallbackTextColor)
-        val message = stringProvider.getRemoteString(titleStringKey)
+        val message = stringProvider.getString(titleStringKey)
```

Comment:
I think we should even use `localeProvider` from constructor instead of `stringProvider`. 🤔

Reply:
![image](https://github.com/user-attachments/assets/8c59b81b-2463-4b11-b5e5-2269436b809c)

will rename to `localeProvider`

Reply:
fixed - [c170b19](https://github.com/salemove/android-sdk-widgets/pull/1210/commits/c170b190292d0e7ac7895f1dda11e8146baa7b55)